src/plugins/intel_cpu/src/mkldnn_node.cpp

```diff
@@ -1076,11 +1076,27 @@ void MKLDNNNode::setDynamicBatchLim(int lim) {
 void MKLDNNNode::appendPostOpArgs(const mkldnn::primitive_attr& attr,
                                   std::unordered_map<int, mkldnn::memory>& primArgs,
                                   const std::vector<MKLDNNMemoryPtr>& postOpsArgs) {
+    constexpr size_t maxPrimArgsCapacity = 32;
```

Comment:
NIT. Shouldn't we import this limitation from the oneDNN directly to be sync with them?